### PR TITLE
fix: retry logic does not work

### LINKED ISSUE #528: Retry is needed.
Log:

```
Stage: Code checkout status: start
exit status 128Stage: Code checkout status: fail with error: exit status 128
Code checkout fails : exit status 128
```

code checkout sometime block due to network instability，retry is needed.

/kind enhancement
/cc @zhujian7 @supereagle

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #528 

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @bbbmj 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhujian7** [comment]
image

File: pkg/event/manager.go [hunk 1]
```diff
@@ -74,8 +74,9 @@ func (em *eventManager) HandleEvent(event *api.Event) error {
 	err := createWorkerForEvent(event)
 	if err != nil {
 		if cloud.IsAllCloudsBusyErr(err) && event.Retry < maxRetry {
-			log.Info("All system worker are busy, wait for 10 seconds")
-			event.Retry++
+			log.Infof("All system worker are busy, wait for 10 seconds, event id:%v retry times:%v",
+				event.ID, event.Retry)
```
Comment (supereagle) [inline pkg/event/manager.go]: No need `event.Retry++` ?

Comment (supereagle) [reply pkg/event/manager.go]: `ResetEvent()` increases retry. Refer to: https://github.com/caicloud/cyclone/blob/master/pkg/store/event.go#L107

**Pull request merged**